tools/src/analyzer/Main.hs

```diff
@@ -36,7 +37,7 @@ main = do
 
     case txFile of
         Nothing   -> pure ()
-        Just file -> analyzeVerifyTimes file confirmationParam logs
+        Just file -> giveStaticConsts $ analyzeVerifyTimes file confirmationParam logs
```

Comment:
Preferably, use `give` as close to the top level as possible (in this case, `main = giveStaticConsts $ do`), because any time you do it in a nested expression, I start to worry it might be a second `give`.